src/Microsoft.ML.PipelineInference/GenerateSweepCandidatesCommand.cs

```diff
@@ -10,23 +10,23 @@
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Internal.Utilities;
-using Microsoft.ML.Runtime.MLTesting.Inference;
 using Microsoft.ML.Runtime.PipelineInference;
 using Microsoft.ML.Runtime.Sweeper;
 
 [assembly: LoadableClass(typeof(GenerateSweepCandidatesCommand), typeof(GenerateSweepCandidatesCommand.Arguments), typeof(SignatureCommand),
     "Generate Experiment Candidates", "GenerateSweepCandidates", DocName = "command/GenerateSweepCandidates.md")]
 
-namespace Microsoft.ML.Runtime.MLTesting.Inference
+namespace Microsoft.ML.Runtime.PipelineInference
 {
     /// <summary>
     /// This is a command that takes as an input:
     /// 1- the schema of a dataset, in the format produced by InferSchema
-    /// 2- the path to the datafile 
-    /// and generates experiment candidates by combining all the transform recipes suggested for the dataset, with all the learners available for the task. 
+    /// 2- the path to the datafile
+    /// and generates experiment candidates by combining all the transform recipes suggested for the dataset, with all the learners available for the task.
     /// </summary>
-    public sealed class GenerateSweepCandidatesCommand : ICommand
+    internal sealed class GenerateSweepCandidatesCommand : ICommand
```

Comment:
We moved this from `test` to `src`.  Is that expected? This command was only needed in tests before....